bruno/collections/Rafiki/Examples/Open Payments/Create Quote.bru

```diff
@@ -5,7 +5,7 @@ meta {
 }
 
 post {
-  url: {{senderOpenPaymentsHost}}/quotes
+  url: {{senderOpenPaymentsHost}}/{{senderTenantId}}/quotes
```

Comment:
Since https://github.com/interledger/rafiki/pull/3152 is merged in, you should be able to remove the explicit inclusion of the `tenantId` in the URL - `(sender/receiver)OpenPaymentsHost` should include the `tenantId` already. Just need to add [this change](https://github.com/interledger/rafiki/pull/3187/files#diff-8220881a210fdf080175cb42a4630e98aff572e5984ced78c0b902c1c828f4ef) to the post-request scripts for getting wallet addresses